### PR TITLE
Default backgrounds

### PR DESCRIPTION
**Pull request by nickine**
#### What does this PR do? (please provide any background)
Aligns the default panels across the shortbreaks platforms to be consistently either white or offwhite.  

#### How can this be tested?
`cd pattern-library && npm link ../brands/alton-tower`
`cd pattern-library && npm link ../brands/paultons`
`cd pattern-library && npm link ../brands/peppa`
`cd pattern-library && npm link ../brands/thorpe`

#### Any tech debt?
no

#### Screenshots / Screencast
peppa-after
peppa-before
paultons-after
paultons-before
alton-after
alton-before
thorpe-after
thorpe-before

#### What gif best describes how you feel about this work?
dumb-dumber-mockingbird-o

- [X] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/alton-towers/src/less/variables.less [hunk 1]
```diff
@@ -316,14 +316,14 @@
...
-@bg-default:               fade(@brand-tertiary-light, 60%);
+@bg-default:               @bg-inverse;
 @bg-muted:                 rgba(255, 255, 255, 0.2);
 @bg-muted-inverse:         rgba(0, 0, 0, 0.1);
 @bg-muted-inverse-normal:  @bg-inverse;
 @bg-inverse:               @color-white;
 @bg-brand:                 @brand-primary-lighter;
 @bg-brand-image:           'https://az772917.vo.msecnd.net/azure/media/2220/hotel-default.jpg';
-@bg-primary:               @bg-inverse;
+@bg-primary:               fade(@brand-tertiary-light, 60%);
```
Comment (jordanhillier90) [inline brands/alton-towers/src/less/variables.less]: primary = tertiary?

File: brands/alton-towers/src/less/variables.less [hunk 2]
```diff
@@ -316,14 +316,14 @@
...
 //##
 
-@bg-default:               fade(@brand-tertiary-light, 60%);
+@bg-default:               @bg-inverse;
 @bg-muted:                 rgba(255, 255, 255, 0.2);
 @bg-muted-inverse:         rgba(0, 0, 0, 0.1);
 @bg-muted-inverse-normal:  @bg-inverse;
 @bg-inverse:               @color-white;
 @bg-brand:                 @brand-primary-lighter;
 @bg-brand-image:           'https://az772917.vo.msecnd.net/azure/media/2220/hotel-default.jpg';
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/variables.less]: this is not your pr, but we should only reference our assets

File: brands/alton-towers/src/less/variables.less (hunk 1 above)
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/variables.less]: this does seem odd

Comment (nickine) [inline brands/alton-towers/src/less/variables.less]: I've made a change to this but the colors file is where you will see the solution to get around it.

File: brands/alton-towers/src/less/variables.less (hunk 2 above)
Comment (nickine) [inline brands/alton-towers/src/less/variables.less]: This image does not exist anywhere on SBP, I'm trying to establish if it is even used on MMB.

**Comment by nickine** [comment]
@jordanhillier90 @jodiedoubleday - have a look at the colours file at how I've got around the tertiary treatments.  problem is that tertiary is so heavily embedded across the rest of the brand that I've had no alternative but to abstract the colour assignment out from the tertiary reference. thought on a postcard?

File: brands/alton-towers/src/less/variables.less (hunk 2 above)
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/variables.less]: i think maybe put it in assets on brands.. if you need help with that as @elliottcrush

**Comment by nickine** [comment]
Holding on merging as we need to have further discussions about what mapping will be affected on live SBP

**Pull request merged**